app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -29,6 +29,7 @@ def create_with_dynflow
             @taxonomy = Organization.new(resource_params)
             sync_task(::Actions::Katello::Organization::Create, @taxonomy)
             @taxonomy.reload
+            session[:organization_id] = @taxonomy.id
```

Comment:
It's only part of the code in `TaxonomyController#select`. IMHO it would be better to redirect to the right action in the controller, and let the controller take care of this action.